### DISCUSSION TITLE
Query Rest to Validate Forms Fields

### DISCUSSION THREAD

**Discussion by angelwine64** [Budibase/budibase#3972] [Ideas]
Hi,

i am rookie budibase's user.

I try to validate some Forms fields by calling Query Rest , with a VALIDATE(button)
then activate the (SAVE button) depending of the query result !!!  Before saving it into database.

Is-it possible to do that way ?  I dont see any doc to point this !

my call Query is OK, but i cant achieve to retrieve the response in my forms !

Can someone guide me,  thank's !

Daniel

**Comment by mjashanks** [comment]
Hi @angelwine64 - I understand what you mean - it is not possible at the moment. But is a great suggestion that I think we should implement.

**Comment by drymek** [comment]
@mjashanks Is it possible with SQL Query or JavaScript code?

**Reply by mjashanks** [reply]
Actually @drymek - you're right - thanks.

You should be able to achieve this with Javascript. You can set conditions on the Button to...

`Update Setting > Disabled > to > when (Javascript Function) > Equals true`

![image](https://user-images.githubusercontent.com/3524181/148930465-6756b4e6-8080-416a-9de2-579bfb55394f.png)

Thanks for pointing it out!
